index.html

```diff
@@ -26,6 +26,19 @@
       <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
+	
+	<script>
+		function get_action(form) {
+
+		var v = grecaptcha.getResponse();
+		if(v.length == 0)
+		{
+			document.getElementById('captcha').innerHTML="You left the Captcha Code empty";
```

Comment:
I have deleted captcha_code_file.php
For now I have commented out the validation js I'll try to look for some other way of doing that